pkg/cmd/install.go

```diff
@@ -255,6 +257,12 @@ func runInstall(args []string, client *action.Install, valueOpts *values.Options
 		return nil, err
 	}
 
+	if client.MaxChartSize > 0 {
+		loader.MaxDecompressedChartSize = client.MaxChartSize
+	}
+	if client.MaxFileSize > 0 {
+		loader.MaxDecompressedFileSize = client.MaxFileSize
```

Comment:
SDK users can run multiple Helm releases concurrently, this will introduce a data race.